.circleci/config.yml

```diff
@@ -13,6 +13,10 @@
        - checkout
        - run: mkdir build build/reports build/dist
 
+       - run:
```

Comment:
I have added this to our docker image. Please remove it here and let me know in case you have any issues.

Reply:
Yep, looks like we don't need this anymore...thanks.  I've removed it from the config.yml